Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -37,22 +40,9 @@ namespace Google.Apis.Tests.Apis.Download
     class MediaDownloaderTest
     {
         /// <summary>The content the "server" uses to send to the client.</summary>
-        static string MediaContent = @"Media content goes here. This is an example of test content.";
+        const string MediaContent = @"Media content goes here. This is an example of test content.";

```

Comment:
Can we replace this with:
```
const bytes[] MediaContent =
    Encoding.UTF8.GetBytes("This string is irrelevant; we just need some binary data.");
```
?

I spent a few seconds wondering whether it mattered that Encoding.UTF8 prepends a BOM by default. It doesn't, because we never actually care about the string. In the one place where we refer to MediaContent directly, we re-encode it as UTF8.

Then we can just use MediaContent.Length everywhere.